ui/components/User.js

```diff
@@ -212,4 +204,20 @@ const User = (props) => {
   );
 };
 
-export default User;
+const UserProvider = (props) => {
+  return (
+    <Provider store={store}>
+      <User {...props} />
+    </Provider>
+  );
+};
+
+const mapDispatchToProps = (dispatch) => ({
+  updateUser: bindActionCreators(updateUser, dispatch),
+});
+
+const mapStateToProps = (state) => ({
+  capabilitiesRegistry: state.get('capabilitiesRegistry'),
+});
+
+export default connect(mapStateToProps, mapDispatchToProps)(UserProvider);
```

Comment:
This PR LGTM now, have  you made sure that you updated component name here in export from "User" to "UserProvider" you also updated same on all the places where this component is used if it is?

Reply:
This PR is on hold for long let's make sure we merge this please sounds off on slack if you made sure of this test case

Reply:
@dottharun other then this were you working asynchronously on other issues also?

Reply:
This is a default export and User component is only used in Header.js. so I don't think we need to change anything else
- Also no other issues atm
Thanks

Reply:
Thankyou

Reply:
I wonder if you might misinterpreted my question here, I was checking code "UserProvider" component is not being used anywhere in codebase so why we even have the component? also I checked header.js in that is uses <User /> component from this file but "User" is not exported instead "UserProvider"?

Checkout header.js - https://github.com/dottharun/meshery/blob/e15ffc079935ececd0692706154d066b00b11e30/ui/components/Header.js#L15

Reply:
Thanks for noting 
- you can check out nearby the end of the section [here](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#description), since it's a "default export" in can be renamed during import without any issue
- This pattern is used at other places too like [here](https://github.com/meshery/meshery/blob/e51ecf569e50c83fcbff0f8e5574691a98d044b6/ui/components/MesherySettings.js#L21)